transports/azure-storage-queues/delayed-delivery.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Azure Storage Queues Delayed Delivery
+component: ASQ
+tags:
+- Azure
+related:
+- persistence/azure-storage/performance-tuning
+reviewed: 2017-04-24
+---
+
+
+In Versions 4.7 and above, the Azure Storage Queues transport no longer relies on the [timeout manager](/nservicebus/messaging/timeout-manager.md) to implement [delayed delivery](/nservicebus/messaging/delayed-delivery.md). Instead, the transport creates infrastructure using Azure Storage account its provided with for queuing which can delay messages using native Storage features.
```

Comment:
@danielmarbach yet, that will make the text simpler, but will omit an important technical detail which should be documented - same storage account used.

Middle ground suggestion:

> Instead, the transport creates infrastructure for queuing which can delay messages using native Storage features. Storage account provided for queuing is used to implement delayed delivery.